GitUI/CommandsDialogs/BrowseDialog/DashboardControl/Dashboard.cs

```diff
@@ -179,7 +179,7 @@ private void groupLayoutPanel_DragDrop(object sender, DragEventArgs e)
                         }
                     }
 
-                    RepositoryManager.AddMostRecentRepository(module.WorkingDir);
+                    RepositoryManager.AddMostRecentRepositoryAsync(module.WorkingDir);
```

Comment:
❗️ This needs to be wrapped in `ThreadHelper.JoinableTaskFactory.RunAsync` and use `FileAndForget`

:memo: Applies to all calls to this method where the caller is not `async` *or* the result of the call does not use `await`

Reply:
yes, I know, thanks. I'm still going with the update. 
...And this class will be retired at the end of the refactor...